spock-specs/src/test/groovy/org/spockframework/smoke/extension/IgnoreIfExtension.groovy

```diff
@@ -162,4 +163,38 @@ class Bar extends Closure {
     ExtensionException ee = thrown()
     ee.message == 'Failed to instantiate condition'
   }
+
+  @IgnoreIf({ true })
+  @IgnoreIf({ false })
+  def "feature is ignored if at least one IgnoreIf annotation is true"() {
+    expect: false
+  }
+
+  @IgnoreIf({ false })
+  @IgnoreIf({ false })
+  @FailsWith(ConditionNotSatisfiedError)
```

Comment:
Would this be a reliable test? If the condition there is an error an the condition always returns true, then this would just always be ignored. IMHO we would need a runner that tests for skipped or successful tests.

Reply:
good catch
actually testing for `ConditionNotSatisfiedError` is enough though imho